locale/en/about/working-groups.md

```diff
@@ -157,27 +201,41 @@ Their responsibilities are:
 * Publishing regular update summaries and other promotional
 content.
 
+### [HTTP](https://github.com/nodejs/http)
+
+The HTTP working group is chartered for the support and improvement of the
+HTTP implementation in Node. It's responsibilities are:
+
+* Addressing HTTP issues on the Node.js issue tracker.
+* Authoring and editing HTTP documentation within the Node.js project.
+* Reviewing changes to HTTP functionality within the Node.js project.
+* Working with the ecosystem of HTTP related module developers to evolve the
+  HTTP implementation and APIs in core.
+* Advising the CTC on all HTTP related issues and discussions.
+* Messaging about the future of HTTP to give the community advance notice of
+  changes.
 
 ### [Roadmap](https://github.com/nodejs/roadmap)
 
 The roadmap working group is responsible for user community outreach
-and the translation of their concerns into a plan of action.
+and the translation of their concerns into a plan of action for Node.js.
 
-The final [ROADMAP](https://github.com/nodejs/node/blob/master/ROADMAP.md) document is still owned by the TC and requires the same approval for changes as any other project asset.
+The final [ROADMAP](./ROADMAP.md) document is still owned by the TC and requires
+the same approval for changes as any other project asset.
 
 Their responsibilities are:
 * Attract and summarize user community needs and feedback.
 * Find or potentially create tools that allow for broader participation.
-* Create Pull Requests for relevant changes to [Roadmap.md](https://github.com/nodejs/node/blob/master/ROADMAP.md)
+* Create Pull Requests for relevant changes to [Roadmap.md](./ROADMAP.md)
 
 
-### [Docker](https://github.com/nodejs/docker-node)
+### [Docker](https://github.com/nodejs/docker-iojs)
 
 The Docker working group's purpose is to build, maintain, and improve official
-Docker images.
+Docker images for the `Node.js` project.
```

Comment:
Good question...? That'll be something to take up with the CTC's version of the info.